### DISCUSSION TITLE
Cant see GPU power limits

### DISCUSSION THREAD

**Discussion by SoulBreacher** [seerge/g-helper#3176] [General]
was looking for a way to still access mow power settings since i cant seem the to download AC from store or the online installer ....the offline one just hangs and does absolutely nothing

i found g helper and it does most of the thing s AC does but i cant change my power limits of gpu just cpu

any help?

i use a 2021 A15 with R7 4800H and RTX 3050
![image_2024-09-26_155724890](https://github.com/user-attachments/assets/cf6501de-f239-4bd5-ac7c-5d92784a0973)

**Comment by seerge** [comment]
@SoulBreacher hello, you are not supposed to see them. Your device don't support it (**as you never had that in Armoury Crate as well**).

You can set a GPU clock limit that indirectly will affect GPU power consumption
